library/core/src/mem/maybe_uninit.rs

```diff
@@ -905,7 +905,7 @@ impl<T> MaybeUninit<T> {
     /// };
     /// ```
     #[stable(feature = "maybe_uninit_ref", since = "1.55.0")]
-    #[rustc_const_unstable(feature = "const_maybe_uninit_assume_init", issue = "none")]
+    #[rustc_const_unstable(feature = "const_maybe_uninit_assume_init_mut", issue = "none")]
```

Comment:
The two changes in this file are the only ones where an unstable feature gate is being changed. Is there a reason this is the case, as opposed to changing the gate for the already-stabilized part of this feature?

Reply:
No reason. I didn't put a lot of thought into the required name changes but I would be happy to consider suggestions.

Reply:
I think it would make more sense to change the gate that's already stabilized, as that would avoid unnecessary breakage on Rust's end.

Reply:
Fixed in 9479358dda11c677064b89d676a8b67d0181be3b